src/fmu/dataio/export/rms/inplace_volumes.py

```diff
@@ -124,6 +124,43 @@ def _convert_table_from_rms_to_legacy_format(table: pd.DataFrame) -> pd.DataFram
             columns="REAL", errors="ignore"
         )
 
+    @staticmethod
+    def _compute_water_zone_volumes_from_totals(table: pd.DataFrame) -> pd.DataFrame:
+        """
+        Calculate 'water' zone volumes by subtracting HC-zone volumes from 'Total'
+        volumes which represents the entire zone. Total volumes are removed after
+        'water' zone volumes have been added to the table.
+        """
+        _logger.debug("Computing water volumes from Totals...")
+
+        total_suffix = "_TOTAL"
+        total_columns = [col for col in table.columns if col.endswith(total_suffix)]
+
+        if not total_columns:
+            raise RuntimeError(
+                "Found no 'Totals' volumes in the table. Please ensure 'Totals' "
+                "are reported and rerun the volumetric job before export."
+            )
```

Comment:
This can also be captured more upfront by evaluation the job settings which already are read

(this example from Drogon)

```
myjob = rmsjobs.Job.get_job(
    owner=["Grid models", "Geogrid", "Grid"],
    type="Volumetrics",
    name="geogrid_volumes",
)

print("USE TOTALS", mydict["Output"][0]["UseTotals"])
```

Reply:
Yes I know, did initially check several things on the job settings e.g. required calculations.. but found it difficult to write tests, had to mock so much. I'll keep it here for now and we can move later if we want 👍